extension-impl/remoting-resteasy/src/main/java/com/alipay/sofa/rpc/server/rest/RestServer.java

```diff
@@ -109,16 +109,16 @@ private void registerProvider(ResteasyProviderFactory providerFactory) {
         }
 
         // 注册cors filter
-        final CorsFilter corsFilter = new CorsFilter();
         Map<String, String> parameters = serverConfig.getParameters();
         if (CommonUtils.isNotEmpty(parameters)) {
+            final CorsFilter corsFilter = new CorsFilter();
             String crossDomainStr = parameters.get(RpcConstants.ALLOWED_ORIGINS);
             String[] domains = StringUtils.splitWithCommaOrSemicolon(crossDomainStr);
             for (String allowDomain : domains) {
                 corsFilter.getAllowedOrigins().add(allowDomain);
             }
+            JAXRSProviderManager.registerCustomProviderInstance(corsFilter);
```

Comment:
I think it is better to register `corsFilter` when crossDomainStr is not empty.

Reply:
ok